doc/ReleaseNotes/ReleaseNotes.tex

```diff
@@ -159,14 +159,15 @@ \section{History}
 	\underline{NEW FUNCTIONALITY}
 	\begin{itemize}
 	        \item --  
+	        \item A new option for printing water contents to a dedicated output file has been added to UZF.  To activate, the keyword WATER_CONTENT is added to the OPTIONS block of UZF, followed by FILEOUT, followed by the user-specified output file name, for example ``water-content.uzf.bin''.  The approach is analogous to the STAGE option within the SFR options block.  Contents of the new file will be written in binary and can be read using flopy's binaryfile utility.  A new test, located in the autotest directory and titled test_gwf_uzf_wc_output.py demonstrates how to read the new binary water content output file.
 	        \item The residual balance error for groundwater flow and solute transport is now written to the diagonal position of the flowja array.  The flowja array is optionally written to the binary model budget file according to user settings in the output control file and other package input files.
 	\end{itemize}
 	
 	\underline{EXAMPLES}
 	\begin{itemize}
 	        \item Added the following new examples: 
 	        \begin{itemize}
-	          \item --
+	          \item -- A new test compares MODFLOW~6 water contents saved with the new WATER_CONTENT keyword in the OPTIONS block of UZF to water contents calculated by MF-NWT and written to the linker file (MT3D-USGS) by the LMT package.  The name of the new test is test_gwf_uzf_wc_output.py
```

Comment:
This section is for new examples (from the modflow6-examples repo), not new autotests.

Reply:
I have moved some of this material up to the NEW FUNCTIONALITY section.  Fixed with f0b161b.  I now understand this section to be where the Hecht-Mendez problem would need to be listed...